airbyte-integrations/connectors/source-salesloft/source_salesloft/schemas/call_dispositions.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
```

Comment:
Can you and additionalProperties and set it to true? https://docs.airbyte.com/connector-development/schema-reference/